presto-testing-docker/src/main/java/com/facebook/presto/testing/containers/NessieContainer.java

```diff
@@ -27,7 +27,7 @@
 {
     private static final Logger log = Logger.get(NessieContainer.class);
 
-    public static final String DEFAULT_IMAGE = "projectnessie/nessie:0.30.0";
+    public static final String DEFAULT_IMAGE = "projectnessie/nessie:0.43.0";
```

Comment:
this is just aligning the nessie version with the one being used in the `presto-iceberg` pom